sections/FTC_Special_LT.tex

```diff
@@ -23,5 +23,10 @@
 			\text{Laplace transform}&e^{-st} = e^{-\sigma t} e^{-i\omega t}
 		\end{array}\right.
 	\end{equation*}
-	This suggests that that the FTC is just a special case of the LT, specifically when $\sigma$ is equal to zero for the Laplace transform of a causal signal. Note that this is not always true, even if the above stated conditions are met. To understand why this is, we need to consider the fact that these integrals are improper. They may converge or diverge if evaluated to infinity, meaning we can not simply say these are always equal.
-\end{document}
\ No newline at end of file
+	This suggests that that the FTC is just a special case of the LT, specifically when $\sigma$ is equal to zero for the Laplace transform of a causal signal. Note that this is not always true, even if the above stated conditions are met. To understand why this is, we need to consider the fact that these integrals are improper. They may converge or diverge if evaluated to infinity, meaning we can not simply say these are always equal. 
+We can say they are equal when the following condition is met:
+		\begin{equation*}
+			\int_{-\infty}^{+\infty}|f(t)|dt<\infty
```

Comment:
Where did you find this test for divergence? AFAIK this is not correct. If you for example take f(t) to be equal to t^3, the integral would evaluate to 0. Even though t^3 actually diverges at \infty.

Reply:
If you look at the note, you can find it actually diverges:
![image](https://user-images.githubusercontent.com/121303742/209536630-730aeb3c-163f-4988-ad41-b99d010f1546.png)

It's also mentioned in the slides that the signal had to be absolutely integrable for them to be equal to each other:
![image](https://user-images.githubusercontent.com/121303742/209538107-4e20609c-2039-46b4-ad15-600d9556111e.png)

Reply:
Ah, I missed the absolute value operation. My bad :P